### PR TITLE
DOCS: Bigger updates to gpercoverseg ref page

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
#

### CODE REVIEW

**Comment by kaknikhil** [comment]
@VMWareJennyBeth Thanks for the PR.

Is there a doc review site where we can review the doc changes made in this PR ?

**Comment by VMWareJennyBeth** [comment]
You mean staged somewhere? I only have that for people with VPN access inside VMWare, unfortunately.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml [hunk 1]
```diff
@@ -168,10 +159,7 @@ newPort|newDataDirectory</codeblock>
         </plentry>
         <plentry>
           <pt>-o <varname>output_recover_config_file</varname></pt>
-          <pd>Specifies a file name and location to output a sample recovery configuration file. The
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:171]: Add this after first sentence: 

"The output file lists the currently invalid segments and their default recovery location. This file format can be used  with -i option for in-place recovery. "

**Comment by niralis13** [comment]
LGTM :)

File: gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml [hunk 2]
```diff
@@ -138,23 +131,21 @@
...
-&lt;recovery_host_address&gt;|&lt;port&gt;|&lt;data_directory&gt;
-</codeblock><p><b>Comments</b></p><p>Lines
-              beginning with <codeph>#</codeph> are treated as comments and ignored.</p>
-            <p><b>Segments to Recover</b></p><p>Each line after the first specifies a segment to
-              recover. This line can have one of two formats. In the event of in-place recovery,
-              enter one group of colon delimited fields in the line. For example:</p><p>
+          <pd>Specifies the name of a file with the details about failed segments to recover.
+              <p>Each line in the config file specifies a segment to recover. This line can have one
+              of two formats. In the event of in-place (incremental) recovery, enter one group of
+              colon delimited fields in the line. For example:</p><p>
```
[the remark this answers is not in the document]
Comment (kaknikhil) [reply gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:137]: @niralis13 
1. I think we should mention that as long as group 2 has a value, we will perform full recovery.
2. Also not sure if we should mention that both the groups can have the same segment repeated. (maybe this is not as important)

Thoughts?

[the remark this answers is not in the document]
Comment (niralis13) [reply gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:137]: @kaknikhil The clarification between full and incremental with recovery_conf is explained in scenario page <which is does not exist at the moment but will be created soon>.

Once we have scenarios page we can put a link here to refer user to specific use case and steps on scenarios page. 

I can you show you the scenario content over zoom.

[the remark this answers is not in the document]
Comment (VMWareJennyBeth) [reply gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:137]: @kaknikhil Just to underline what Nirali is saying, we are trying to move as much explanatory narrative as possible out of the gprecoverseg reference page and into what will be two new narrative topics (one is the Scenarios page Nirali is referring to). We link to these new pages as necessary from what we hope will be a more streamlined gprecoverseg reference page.

[the remark this answers is not in the document]
Comment (kaknikhil) [reply gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:137]: @Nirali13 @VMWareJennyBeth 

That makes sense, thanks. 

In that case, the only actionable comment is https://github.com/greenplum-db/gpdb/pull/12462#discussion_r699617759

[the remark this answers is not in the document]
Comment (VMWareJennyBeth) [reply gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml]: Fixed and committed.

**Review by kaknikhil — APPROVED** [review]

**Review by dyozie — APPROVED** [review]

File: gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml [hunk 3]
```diff
@@ -151,7 +151,9 @@ newPort|newDataDirectory</codeblock>
             <codeblock>sdw1-1|50001|/data1/mirror/gpseg16<b>SPACE</b>sdw4-1|50001|/data1/recover1/gpseg16</codeblock>
             <p><b>Obtaining a Sample File</b></p>
             <p>You can use the <codeph>-o</codeph> option to output a sample recovery configuration
-              file to use as a starting point.</p></pd>
+              file to use as a starting point. The output file lists the currently invalid segments
+              and their default recovery location. This file format can be used with
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:155]: with -> with the

Comment (VMWareJennyBeth) [reply gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml]: Done!

**Pull request merged**